doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,46 @@
+# User Guides for the generic Virtual Schema for Document Files
```

Comment:
```suggestion
# User Guides for the Generic Virtual Schema for Document Files
```

Reply:
This header breaks my brain somehow :D Can we rephrase it somehow?

Reply:
```suggestion
# User Guides for the Generic Document Files Virtual Schema
```

Reply:
:+1: